ibc/Cargo.toml

```diff
@@ -67,3 +67,8 @@ parity-scale-codec = [
   "ibc-core-host-cosmos/parity-scale-codec",
   "ibc-primitives/parity-scale-codec",
 ]
+arbitrary = [
+  "ibc-apps/arbitrary",
+  "ibc-core/arbitrary",
```

Comment:
clients, core-host-cosmos are missing.

Reply:
`clients` doesn't have it, but I've added it to `core-host-cosmos` to propagate to its deps